pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminBuilderImpl.java

```diff
@@ -54,13 +55,23 @@ private PulsarAdminBuilderImpl(ClientConfigurationData conf) {
 
     @Override
     public PulsarAdminBuilder clone() {
-        return new PulsarAdminBuilderImpl(conf.clone());
+        PulsarAdminBuilderImpl pulsarAdminBuilder = new PulsarAdminBuilderImpl(conf.clone());
```

Comment:
If we have this line, why do we need `pulsarAdminBuilder.gzipCompressionEnabled = gzipCompressionEnabled;`?

Reply:
that is to implement the cloning. the clone method should return a similar object.

Reply:
It's needed here. I'm not changing the original implementation and how it makes a deep clone. I don't see a need to make any additional changes.